redis.conf

```diff
@@ -498,12 +498,24 @@ slave-priority 100
 #
 # Please check http://redis.io/topics/persistence for more information.
 
-appendonly no
+appendonly yes
 
 # The name of the append only file (default: "appendonly.aof")
 
 appendfilename "appendonly.aof"
 
+# AOF will be written to NVML's PMEMlog pool. NOTICE: when aof-use-nvml is
+# enabled appendfsync option will be ignored.
+aof-use-nvml yes
+
+# When enabled all writes to the log will be immediate, no write buffer
+# will be used.
+aof-nvml-direct yes
```

Comment:
There should be no configuration flag for that - "direct" mode should be one and only option if NVML is in use.